packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -26,13 +26,59 @@ t d
 					<!-- Mapping section -->
 					<div class="form-section">
 						<h5 class="mb-1">Mapping</h5>
-						<p class="mb-2">Map model variables to dataset columns. Don't forget the timeline variable.</p>
+						<p class="mb-2">
+							Select a subset of output variables of the model and individually assosiate them to columns in the
+							dataset.
+						</p>
+
+						<!-- Mapping table: Time variables -->
+						<DataTable
+							v-if="mapping.find((ele) => ele.modelVariable === 'timestamp')"
+							class="mapping-table"
+							:value="mapping.filter((ele) => ele.modelVariable === 'timestamp')"
+						>
+							<Column field="modelVariable">
+								<template #header>
+									<span class="column-header">Model: Timeline variable</span>
+								</template>
+								<template #body="{ data, field }">
+									<Dropdown
+										class="w-full"
+										:placeholder="mappingDropdownPlaceholder"
+										v-model="data[field]"
+										:disabled="true"
+										:options="modelStateOptions?.map((ele) => ele.referenceId ?? ele.id)"
+									/>
+								</template>
+							</Column>
+							<Column field="datasetVariable">
+								<template #header>
+									<span class="column-header">Dataset: Timeline variable</span>
+								</template>
+								<template #body="{ data, field }">
+									<Dropdown
+										class="w-full"
+										:placeholder="mappingDropdownPlaceholder"
+										v-model="data[field]"
+										:options="datasetColumns?.map((ele) => ele.name)"
+									/>
+								</template>
+							</Column>
+							<Column field="deleteRow">
+								<template #header>
+									<span class="column-header"></span>
+								</template>
+								<template #body="{ data }">
+									<Button class="p-button-sm p-button-text" icon="pi pi-trash" @click="deleteMapRow(data.id)" />
```

Comment:
I don't think time should have a delete button, designs don't have it

Reply:
As this table is just all of the fields that have been selected as `timestamp` on the model side it seems helpful that a user can delete these.
If a user ends up selecting `timestamp` then they may end up in a state which they want to correct it but can only do so via recreating the node
<img width="462" alt="image" src="https://github.com/user-attachments/assets/c42ffcd1-e4f5-458a-922d-0e9da394af0d">

If I do not allow the user to select the `timestamp` in this dropdown to avoid this exact scenario then we may run into edge cases that either do or do not exist yet so I think it is worth keeping to give the user more control